### ISSUE TITLE
Guild points unobtainable

### ISSUE THREAD

**Issue by JinxDojo** [DerpyProjectGroup/topaz#1897]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened, and I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)

## Branch affected by issue 

`base`
(2022/05/25)

## Steps to reproduce

Currently, when inquiring at a guild about how many points can be earned, the message returned is that "You are not eligible to receive guild points at this time." (This is accurate: it won't accept trades. However, I've earned no guild points for that day--and it is, indeed, my main/assigned guild. This is persistent across multiple characters and multiple guilds.)

## Expected behavior

I'd like to be able to earn guild points again. :)

**Comment by UynGH** [comment]
Hello, thank you for reporting this issue.

It was already reported here: https://github.com/LandSandBoat/server/issues/1846 and fixed here: https://github.com/LandSandBoat/server/pull/1848.

I encourage you to fully update to the latest version before reporting an issue.

Thank you!

**Comment by JinxDojo** [comment]
> Hello, thank you for reporting this issue.
> 
> It was already reported here: #1846 and fixed here: #1848.
> 
> I encourage you to fully update to the latest version before reporting an issue.
> 
> Thank you!

Ah, thank you for linking those. I did not see them. I had talked to TeoTwawki a few days back and made resolutions to report particular issues--but I see this one is already resolved.

Unfortunately, I am not the server administrator, so "fully updat[ing]" is not possible. Still, I hope I'll be able to be of service in identifying/fixing some of the issues.
